sct-commons/src/main/java/org/lfenergy/compas/sct/commons/util/Utils.java

```diff
@@ -366,4 +327,14 @@ public static <T> T copySclElement(T object, Class<T> clazz) {
         }
     }
 
+    /**
+     * Extract content of P element of given type
+     *
+     * @param type type attribute of P element
+     * @param listOfP list of P elements
+     * @return content of P element if P element with given type exist, else empty Optional.
```

Comment:
return VALUE of THE FIRST of TYPE element in listOfP if... ?

Reply:
OK